edx_api/grades/__init__.py

```diff
@@ -103,7 +100,7 @@ def get_course_current_grades(self, course_id):
                 resp = self.requester.get(resp_json['next'])
                 resp.raise_for_status()
                 resp_json = resp.json()
-                grade_entries.extend((CurrentGrade(entry) for entry in resp_json["results"]))
+                grade_entries.extend(CurrentGrade(entry) for entry in resp_json["results"])
         else:
```

Comment:
Is there any specific reason for making this change?

Reply:
The python codemod [pyupgrade](https://github.com/asottile/pyupgrade) changed this itself. I think, in previous python versions, a bracket would be required to write such expressions which is not necessary now. Is it breaking anything?